packages/runtime/container-runtime/src/test/containerRuntime.spec.ts

```diff
@@ -1053,8 +1054,8 @@ describe("Runtime", () => {
 			});
 
 			it("process remote op with unrecognized type and 'FailToProcess' compat behavior", async () => {
-				const futureRuntimeMessage: ContainerRuntimeMessage = {
-					type: "FROM_THE_FUTURE" as ContainerMessageType,
+				const futureRuntimeMessage: UnknownContainerRuntimeMessage = {
```

Comment:
I love this typing!

Reply:
but we don't want that type really doing out; so, it is back to being more or less free for all which is what the test is doing.